### PR TITLE
updated recording branch

### PR DESCRIPTION
**Pull request by alanamramjit**
updated recording infrastructure for #6

---
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/354)

### CODE REVIEW

File: webrender/src/record.rs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+               ref msg @ &ApiMsg::AddRawFont(..) | 
+               ref msg @ &ApiMsg::AddNativeFont(..) |
+               ref msg @ &ApiMsg::AddImage(..) |
+               ref msg @ &ApiMsg::SetRootPipeline(..) |
+               ref msg @ &ApiMsg::UpdateImage(..) |
+               ref msg @ &ApiMsg::Scroll(..)|
+               ref msg @ &ApiMsg::TickScrollingBounce|
+               ref msg @ &ApiMsg::DeleteImage(..)|
+               ref msg @ &ApiMsg::SetRootStackingContext(..) =>{
+                       let filename = format!("record/frame_{}.bin", frame);
```
Comment (glennw) [inline webrender/src/record.rs]: Can this share code with or directly use the write_data function above? (Just so the filename isn't declared in two locations etc).

**Comment by glennw** [comment]
@alanamramjit One minor issue, then it should be good to go.

**Comment by alanamramjit** [comment]
You're right; this should probably create a RecordMsg enum so that it's easier to deserialize anyway.

**Pull request merged**